spec/50-privacy.md

```diff
@@ -24,4 +24,4 @@ Vendors extremely sensitive to personal information exposure MAY implement selec
 
 ## Other risks
 
-When vendors include `traceparent` and `tracestate` headers in responses, these values may inadvertently be passed to cross-origin callers. Vendors should ensure that they include only these response headers when responding to systems that participated in the trace.
+When vendors include `traceparent` and `tracestate` headers in responses, these values may inadvertently be passed to cross-origin callers. Hence, if vendors include this information in responses, they should be aware that it could be seen by systems that did not participate in the trace.
```

Comment:
```suggestion
When vendors include values from the `traceparent` and `tracestate` headers in responses, these values may inadvertently be passed to cross-origin callers. Hence, if vendors include this information in responses, they should be aware that it could be seen by systems that did not participate in the trace.
```